nix/default.nix

```diff
@@ -10,7 +10,13 @@
     { pkgs, lib, ... }:
     let
       self = inputs.self;
-      version = if (self ? rev) then self.rev else "dirty-${self.dirtyRev}";
+      version =
+        if (self ? rev) then
+          self.rev
+        else if (self ? dirtyRev) then
+          "dirty-${self.dirtyRev}"
+        else
+          "(dev)";
```

Comment:
Maybe `(devel)` to get closer to what the default output would be when we are not setting any version at all.

Reply:
Sure, updated